src/pages/Question/CommentItem/CommentItem.tsx

```diff
@@ -136,12 +143,16 @@ const CommentItem: React.FC<CommentProps> = ({
           </div>
         ) : (
           <>
-            <span>{comment.body}</span>
+            <MarkdownViewer
+              className={styles.commentBody}
+              source={comment.body}
+            />
+
             <label>
               <div>–</div>
-              {/*<Link to={`/users/${comment.user.id}`}>*/}
-              <div className={styles.username}>{comment.user.username}</div>
-              {/*</Link>*/}
+              <Link to={`/users/${comment.user.id}`}>
+                <p className={styles.username}>{comment.user.username}</p>
+              </Link>
```

Comment:
`div`로 감싸주는 게 일관될 것 같습니다.
```suggestion
              <div className={styles.username}>
                <Link to={`/users/${comment.user.id}`}>{comment.user.username}</Link>
              </div>
```